boardinfo/de5.json

```diff
@@ -1549,9 +1549,7 @@
         "PCIE_WAKE_n": {
             "IOSTANDARD": "2.5 V", 
             "LOC": "PIN_BD35"
-        }
-    }, 
-    "PLL": {
+        },
         "\"PCIE_REFCLK_p(n)\"": {
```

Comment:
You should not need the escaped quotes here. Does the signal name actually include the quotes?